src/components/list/list.tsx

```diff
@@ -151,10 +151,14 @@ export class List {
         const listItems = this.items.filter(this.isListItem);
```

Comment:
> **How to reproduce:**
> Update the items in a list with items that doesn't have the `selected` property will result in this error:
>
> ![Skärmavbild 2022-01-03 kl  16 27 34](https://user-images.githubusercontent.com/21986391/147948704-7ae8ef0d-311a-48cf-ae4d-8b6af6e12e4b.png)

Ah… this only happens when the list has radio-buttons or checkboxes, right? I was having trouble reproducing this until I examined that screenshot in a little more detail…

Reply:
Yea sorry, should have make that more clear. It only happens for list types that is not multiple, i.e. `radio` and `selectable`.

Reply:
`checkbox` is multiple, so we will take another path for that. That path will result in that we are assigning `this.mdcList.selectedIndex` to an empty list, which seems to be valid (it doesn't throw any errors at least).